picard/util/imageinfo.py

```diff
@@ -106,7 +106,7 @@ def identify(data):
             pass
 
     # PDF
-    elif data[:4] == '%PDF':
+    elif data[:4] == b'%PDF':
```

Comment:
This is actually a bug fix, right? If I see this correctly recognizing PDF was broken. If so I think we should add a ticket for the changelog.

Reply:
Yes, but I wonder when it actually happens, I didn't yet look in details.

Reply:
Found it, it happens at least in one case:
when one drops a pdf on cover art image box
Without the patch, it fails with:
D: 21:07:10,873 ui.coverartbox.dropEvent:99: Dropped file:///home/zas/Téléchargements/t.pdf url (with 0 bytes of data)
W: 21:07:10,873 ui.coverartbox.load_remote_image:458: Can't load image: Unrecognized image data

Reply:
Fix was moved to https://github.com/metabrainz/picard/pull/1494